### ISSUE TITLE
Any advice or document on finetune the VAE?

### ISSUE THREAD

**Issue by VCBE123** [lllyasviel/ControlNet#500]
Just as the question. I stuck in finetuen the VAE model on my costum medical image datasets. I adop the LPIPSWithDiscriminator Loss from [https://huggingface.co/spaces/multimodalart/latentdiffusion/tree/main/latent-diffusion/ldm/modules/losses](https://huggingface.co/spaces/multimodalart/latentdiffusion/tree/main/latent-diffusion/ldm/modules/losses)  but the sample is crupt. 
I also find the issue here [https://github.com/CompVis/stable-diffusion/issues/425](https://github.com/CompVis/stable-diffusion/issues/425) and [https://github.com/CompVis/stable-diffusion/issues/409](https://github.com/CompVis/stable-diffusion/issues/409)

**Comment by keyu-tian** [comment]
@VCBE123 any advance?

**Comment by wtliao** [comment]
@VCBE123 @keyu-tian Hi, I am also trying to do that. Do you have any luck?

**Comment by keyu-tian** [comment]
@wtliao i think im finetuning VQ-VAE well on OpenImages. Just using their very small learning rate (2e-6 for batch size 512) and it works for me. I use the augmentation of resizing-to-384-then-random-crop-to-256 and normalization from [0, 1] to [-1, 1].

**Comment by Z7Gao** [comment]
Hi, it is amazing since you are use bs=512 for 256*256 images. I wonder how large is your gpu memory?

**Comment by keyu-tian** [comment]
> Hi, it is amazing since you are use bs=512 for 256*256 images. I wonder how large is your gpu memory?

oh, i use 128*128 images, and 8 V100s are enough for them

**Comment by qiqigit** [comment]
> > Hi, it is amazing since you are use bs=512 for 256*256 images. I wonder how large is your gpu memory?
> 
> oh, i use 128*128 images, and 8 V100s are enough for them

@keyu-tian 
Would you mind sharing the training code for finetuning Stable Diffusion VAE? I'd really appreciate that!
